README.md

```diff
@@ -75,9 +75,27 @@ To run the tests the following tools need to be installed:
     $ pip3 install --user git+https://github.com/klobuczek/boltkit@1.3#egg=boltkit
     $ neoctrl-install -e 4.0.2 servers
     $ neoctrl-configure servers/neo4j-enterprise-4.0.2 dbms.directories.import= dbms.default_listen_address=::
-    $ neoctrl-set-initial-password password servers/neo4j-enterprise-4.0.2
+    $ neoctrl-set-initial-password pass servers/neo4j-enterprise-4.0.2
```

Comment:
Default password used in driver helper is pass, so updated it here to keep consistency.
https://github.com/neo4jrb/neo4j-ruby-driver/blob/9fa6427c0761d5dfc021a60de5e3f4a61cc2db9b/spec/support/driver_helper.rb#L16